src/components/TodoItem/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import users from '../../api/users';
+import './TodoItem.css';
+
+export default class TodoItem extends Component {
+  state = {
+    usersForTask: users,
+    completed: this.props.completed,
+  }
+
+  toogleCompleted = (e) => {
+    e.preventDefault();
+    this.setState(state => ({ completed: !state.completed }));
+  }
+
+  render() {
+    const { title, userId = 0 } = this.props;
```

Comment:
in what cases userId is 0?
you marked it as required in propTypes, so you can safely remove it from here